test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenManagementSpecs.java

```diff
@@ -267,24 +266,20 @@ public HapiApiSpec kycMgmtSuccessCasesWork() {
 						newKeyNamed("oneKyc"),
 						newKeyNamed("twoKyc"),
 						tokenCreate(withPlusDefaultTrue)
-								.kycDefault(true)
 								.kycKey("oneKyc")
 								.treasury(TOKEN_TREASURY),
 						tokenCreate(withPlusDefaultFalse)
-								.kycDefault(false)
-								.kycKey("twoKyc")
 								.treasury(TOKEN_TREASURY)
 				).when(
-						tokenTransact(
```

Comment:
I think that you can add new success case in which the account `misc` was granted KYC, he can transact and after that, he was revoked KYC -> check that he cannot transact anymore